assets/js/components/dashboard-sharing/ModuleRecoveryAlert/Errors.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * ModuleRecoveryAlert Error component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+
+export default function Errors( { recoveryErrors } ) {
+	return (
+		<div className="googlesitekit-module-recovery-error">
```

Comment:
Minor detail, but let's make the class name plural for consistency with the component but also because it includes a list of errors 😄

Reply:
Sharp spot! Apologies for missing this. Updated.